src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -703,6 +710,21 @@ public void setWaitForPodSec(Integer waitForPodSec) {
         this.waitForPodSec = waitForPodSec;
     }
 
+    @Override
+    public Cloud reconfigure(@NotNull StaplerRequest req, JSONObject form) throws Descriptor.FormException {
+        KubernetesCloud newCloud = (KubernetesCloud) super.reconfigure(req, form);
```

Comment:
in the superclass this was made private in https://github.com/jenkinsci/jenkins/pull/7658/commits/e0944c76fb3d050bd98201f4f77e2283eb989b4c

does it need to be changed to be public?

Reply:
https://github.com/jenkinsci/jenkins/pull/8053